terms.html

```diff
@@ -107,16 +118,26 @@
   <dd>
 A program, such as a storage vault or personal <a>verifiable credential</a>
 wallet, that stores and protects access to <a>holders'</a>
+<a>verifiable credentials</a>.
+  </dd>
+  <dt><dfn data-lt="resources">resource</dfn></dt>
+  <dd>
+Anything, including documents, people, physical objects, and abstract concepts,
+that information can be expressed about (see [RDF primer](https://www.w3.org/TR/rdf11-primer/#introduction)).
+<a>Entities</a> are resources, but the not every resources is an <a>entity</a>.
 <a>verifiable credentials</a>.
   </dd>
   <dt><dfn>selective disclosure</dfn></dt>
   <dd>
 The ability of a <a>holder</a> to make fine-grained decisions about what
 information to share.
   </dd>
-  <dt><dfn data-lt="subjects|subject's">subject</dfn></dt>
-  <dd>
-A thing about which <a>claims</a> are made.
+  <dt><dfn data-lt="subjects|subject's|subject(s)">subject</dfn></dt>
+The <a>resource</a> about which a <a>claim</a> is made.
+Note that the common practice of speaking about "*the* <a>subject</a> of a <a>verifiable credential</a>"
+may be incorrect, as <a>verifiable credentials</a> can contain multiple <a>claims</a> that can be made
+about different <a>resources</a>, in which case *the* <a>subject</a> of that <a>verifiable credential</a>
+is not determinable.
```

Comment:
```suggestion
is indeterminate.
```

Reply:
or
```suggestion
is indeterminable.
```

Reply:
or
```suggestion
cannot be determined.
```

Reply:
This is outside the scope of this PR

Reply:
> This is outside the scope of this PR

In a sense... but it's an integral part of the text you've rewritten in this PR, and a change via this PR is considerably lighter weight than another PR, so I believe it should be handled here. I *strongly* request that you not mark change requests "resolved" when rejecting them until/unless your rejection is acknowledged by the change requester. I would not have seen your rejection here, except that I clicked the "@RieksJ marked this conversation resolved" indicator and noticed that you hadn't acted on the suggestion.

Upon further reflection, I believe that my last suggestion is the best, and reiterate it here:

```suggestion
cannot be determined.
```